_inc/client/static.jsx

```diff
@@ -12,23 +12,24 @@ import store from 'state/redux-store';
 import StaticMain from 'static-main';
 import StaticWarning from 'components/jetpack-notices/static-warning';
 
-window.staticHtml = Server.renderToStaticMarkup(
-	<div>
-		<Provider store={ store }>
-			<StaticMain />
-		</Provider>
-	</div>
-);
+export default () =>
+	renderToStaticMarkup(
+		<div>
+			<Provider store={ store }>
+				<StaticMain />
+			</Provider>
+		</div>
+	);
 
-window.noscriptNotice = Server.renderToStaticMarkup(
+window.noscriptNotice = renderToStaticMarkup(
 	<Provider store={ store }>
 		<noscript>
 			<StaticWarning />
 		</noscript>
 	</Provider>
 );
 
-window.versionNotice = Server.renderToStaticMarkup(
+window.versionNotice = renderToStaticMarkup(
```

Comment:
Will probably need to move `noscriptNotice` and `versionNotice` to separate files,